include/stl2/detail/algorithm/partial_sort_copy.hpp

```diff
@@ -42,8 +42,8 @@ STL2_OPEN_NAMESPACE {
 	{
 		auto r = result_first;
 		if(r != result_last) {
-			std::tie(first, r) =
-				ext::copy(std::move(first), last, std::move(r), result_last);
+			auto [in, out] = ext::copy(std::move(first), last, std::move(r), result_last);
+			std::tie(first, r) = std::tie(in, out);
```

Comment:
Ditto eww `tie` (I'll stop calling these out now and assume you're smart enough to grep for `tie`.)